packages/stryker-webpack-transpiler/src/compiler/ConfigLoader.ts

```diff
@@ -30,14 +31,14 @@ export default class ConfigLoader {
     return webpackConfig;
   }
 
-  private loaderWebpackConfigFromProjectRoot(configFileLocation: string) {
-    try {
-      return this.loader(path.resolve(configFileLocation));
-    } catch {
-      throw new Error(`Could not load webpack config at "${configFileLocation}", file not found.`);
-    }
+private loaderWebpackConfigFromProjectRoot(configFileLocation: string) {
+  if (!fs.existsSync(path.resolve(configFileLocation))) {
```

Comment:
Could we catch `path.resolve(configFileLocation)` in a variable and use that 3 times? I would use the resolved name in the exception message as well (so its really clear)